common-rest/src/main/java/org/uniprot/api/rest/request/HttpServletRequestContentTypeMutator.java

```diff
@@ -67,8 +68,10 @@ static void orderKeysSoPathVariablesLast(List<String> pathVariables) {
     }
 
     public void mutate(MutableHttpServletRequest request) {
-        mutateAcceptHeader(request);
-        mutateAcceptEncodingHeader(request);
+        if (!request.getRequestURI().endsWith(DOWNLOAD_RUN_SUFFIX)) {
```

Comment:
I would prefer this if check to be performed by the caller and this method only does the mutation part.